grails-app/controllers/au/org/emii/aatams/SensorController.groovy

```diff
@@ -32,8 +32,6 @@ class SensorController extends ReportController {
         def tag = tagFactoryService.lookupOrCreate(params.tag)
         assert(tag)
 
-        params.tag = tag
```

Comment:
I feel reluctant to merge this when nobody can state the reason for why it broke or stopped working.

If the code was a front-end only -  then I wouldn't hesitate - since rollback is so trivial in that case.

Someone can merge if they want, @xhoenner or @pmbohm